addons/arsenal/functions/fnc_buttonClearAll.sqf

```diff
@@ -0,0 +1,62 @@
+/*
+ * Author: Alganthe
+ * Clear the current container.
+ *
+ * Arguments:
+ * 0: Arsenal display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display"];
+
+// Clear container
+switch (GVAR(currentLeftPanel)) do {
+    case IDC_buttonUniform: {
+        {GVAR(center) removeItemFromUniform _x} foreach (uniformItems GVAR(center));
+        GVAR(currentItems) set [15, []];
+    };
+    case IDC_buttonVest: {
+        {GVAR(center) removeItemFromVest _x} foreach (vestItems GVAR(center));
+        GVAR(currentItems) set [16, []];
+    };
+    case IDC_buttonBackpack: {
+        {GVAR(center) removeItemFromBackpack _x} foreach (backpackItems GVAR(center));
```

Comment:
https://community.bistudio.com/wiki/clearAllItemsFromBackpack

Reply:
Considering backpackItems returns list of items with duplicates... This might be a forEach loop over thousands of items.
clearAllItemsFromBackpack is a single command call.